cmd/query/app/server.go

```diff
@@ -48,23 +48,23 @@ import (
 // Server runs HTTP, Mux and a grpc server
 type Server struct {
 	logger       *zap.Logger
+	healthCheck  *healthcheck.HealthCheck
```

Comment:
I felt like this was nicer ordering because this felt related to the logger to me since they both come from the service and are both related to "reporting status".